cpp/src/gandiva/precompiled/CMakeLists.txt

```diff
@@ -65,7 +65,7 @@ function(add_precompiled_unit_test REL_TEST_NAME)
   )
   target_compile_definitions(${TEST_NAME} PRIVATE GANDIVA_UNIT_TEST=1)
   add_test(NAME ${TEST_NAME} COMMAND ${TEST_NAME})
-  set_property(TEST ${TEST_NAME} PROPERTY LABELS gandiva;unittest ${TEST_NAME})
+  set_property(TEST ${TEST_NAME} PROPERTY LABELS gandiva-tests {TEST_NAME})
```

Comment:
Why is it "gandiva-tests" rather than "gandiva"?

Reply:
This is the result of https://github.com/apache/arrow/commit/9fcce64e6108dd911c9cfcd4121ea33e2b447c91

Reply:
no, we removed the microbenchmarks from the tests.

Reply:
Look more closely at https://github.com/apache/arrow/commit/9fcce64e6108dd911c9cfcd4121ea33e2b447c91

We now have targets `gandiva` (libraries) `gandiva-tests` (tests) and `gandiva-benchmarks` (benchmarks). The label matches the target name

Reply:
Ah, ok. I guess it doesn't make a difference when running e.g. `ctest -L arrow`.

Reply:
We have to run `ctest -L arrow-tests` now but yeah https://github.com/apache/arrow/blob/master/ci/travis_script_gandiva_cpp.sh#L26